### PR TITLE
feat(google-drive): search for file / folder action

### PR DESCRIPTION
**Pull request by kanarelo**
## What does this PR do?

Fixes # (issue)

ANNOUNCEMENT=true

### CODE REVIEW

**Review by abuaboud — APPROVED** [review]

File: packages/pieces/google-drive/src/lib/action/get-file-by-name.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  name: 'get-file-by-name',
+  displayName: 'Get File By Name',
+  description: 'Get file/folder from the Google Drive root or folder provided',
+  props: {
+    name: Property.ShortText({
+      displayName: 'Name',
+      description: 'Name to search for',
+      required: true,
+    }),
+    folderId: Property.ShortText({
```
Comment (abuaboud) [inline packages/pieces/google-drive/src/lib/action/get-file-by-name.ts]: Can we make this dropdown like the trigger?

**Review by abuaboud — CHANGES_REQUESTED** [review]

File: packages/pieces/google-drive/src/lib/action/get-file-by-name.ts (hunk 1 above)
Comment (kanarelo) [reply packages/pieces/google-drive/src/lib/action/get-file-by-name.ts]: Fixed

**Comment by kanarelo** [comment]
@abuaboud let me know if I should close this. Thanks.

**Comment by kishanprmr** [comment]
Hello @kanarelo ,

PR #3632 has moved to the Community folder. Can you please update your feature branch?

Let me know if you face any issues.

**Comment by kanarelo** [comment]
> Hello @kanarelo ,
> 
> PR #3632 has moved to the Community folder. Can you please update your feature branch?
> 
> Let me know if you face any issues.

Hey Yeah, I am having a couple of build issues, but I will tel you if and when i can build it.

**Review by abuaboud — APPROVED** [review]

**Pull request merged**